src/dmd/func.d

```diff
@@ -2570,9 +2570,8 @@ extern (C++) FuncDeclaration resolveFuncCall(const ref Loc loc, Scope* sc, Dsymb
 
     Match m;
     m.last = MATCH.nomatch;
-
-    const(char)* failMessage;
-    functionResolve(&m, s, loc, sc, tiargs, tthis, fargs, &failMessage);
+    functionResolve(&m, s, loc, sc, tiargs, tthis, fargs, null);
+    auto orig_s = s;
```

Comment:
Why the extra reference? You can just use `s`.

Reply:
@RazvanN7 What about line 2609?
https://github.com/dlang/dmd/pull/8232/files/8facce8f4138910a600f3580dba0279666573e21#diff-ca782fb8b1ce12422a29fed42ffe0cadL2609

Reply:
Oh, ok missed that